docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,511 @@
+its integration policies.
+
+. Click the name of the integration policy to edit it.
+
+. Click the down arrow next to enabled streams, and under *Advanced options*,
+add the processor definition. The processor will be applied to the data set
+where it's defined.
++
+[role="screenshot"]
+image::images/migration-add-processor.png[Screen showing how to add a processor to an integration policy]
++
+For example, the following processor adds geo-specific metadata to host events:
++
+[source,yaml]
+----
+- add_host_metadata:
+    cache.ttl: 5m
+    geo:
+      name: nyc-dc1-rack1
+      location: 40.7128, -74.0060
+      continent_name: North America
+      country_iso_code: US
+      region_name: New York
+      region_iso_code: NY
+      city_name: New York
+----
+
+In {kib}, look at the data again to confirm it contains the fields you expect.
+
+[discrete]
+== Preserve raw events
+
+In some cases, {beats} modules preserve the original, raw event, which consumes
+more storage space, but may be a requirement for compliance or forensic use
+cases.
+
+In {agent}, this behavior is optional and disabled by default.
+
+If you must preserve the raw event, edit the integration policy, and for each
+enabled data stream, click the *Preserve original event* toggle.
+
+[role="screenshot"]
+image::images/migration-preserve-raw-event.png[Screen showing how to add a processor to an integration policy]
+
+Do this for every data stream with a raw event you want to preserve.
+
+[discrete]
+== Migrate custom dashboards
+
+Elastic integration packages provide many assets, such as pre-built dashboards,
+to make it easier for you to visualize your data. In some cases, however, you
+might have custom dashboards you want to migrate.
+
+Because {agent} uses data streams, the fields exported by an {agent} are
+slightly different from those exported {beats}. Any custom dashboards that
+you created for {beats} need to be modified or recreated to use the new fields.
+
+//REVIEWERS: Sounds like Beats is moving to data streams, too. Does this whole
+//section only apply to Beats 7.17 and earlier? Should I mention that now?
+
+You have a couple of options for migrating custom dashboards:
+
+* (Recommended) Recreate the custom dashboards based on the new data streams.
+
+* <<create-index-aliases,Create index aliases to point to the new data streams>>
+and continue using custom dashboards.
+
+[discrete]
+[[create-index-aliases]]
+=== Create index aliases to point to data streams
+
+You may want to continue using older custom dashboards if the dashboards
+installed with an integration are not adequate. To do this, use index aliases
+to feed data streams into your existing custom dashboards.
+ 
+For example, custom dashboards that point to `filebeat-` or `metricbeat-` can be
+aliased to use the equivalent data streams, `logs-` and `metrics-`.
+
+To use aliases:
+
+. Add a `filebeat` alias to the `logs-` data stream. For example:
```

Comment:
This section needs more work. I thought it was working when I tested it yesterday, but starting in a fresh environment (without having written any filebeat or metricbeat data), I don't see any metrics in my dashboards after creating the aliases.

Reply:
Actually...I think I figured this out. I think the version info is required in the alias.